feat: add ecommerce federated dataset (#12)
algolia/datasets: ecommerce-federated/README.md

# Ecommerce Federated

> The files in this folder contain products (with query suggestions), articles, and FAQ records as you would see in a federated search experience.

Reviewer: ```suggestion
This dataset collection contains products (with [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/react/)), articles, and FAQ records as you would see in a federated search experience.
```

> 
> ## Live demo

Reviewer: ```suggestion
```

> You can see a live demo of this dataset [here](https://codesandbox.io/s/github/algolia/autocomplete/tree/next/examples/two-column-layout).

Reviewer: ```suggestion
You can see a live demo using this dataset [here](https://codesandbox.io/s/github/algolia/autocomplete/tree/next/examples/two-column-layout).
```

> You can see a live demo of this dataset [here](https://codesandbox.io/s/github/algolia/autocomplete/tree/next/examples/two-column-layout).

Reviewer 2: I'd suggest we also add a link to the guide(s) which use this collection of dataset, so anyone visiting this page can further explore what they can build using those. 

> and learn how those dataset are put in motion in the "Ecommerce federated guide"

## Structure

### Products index

> This index contains **fashion products**.

Reviewer: ```suggestion
This index contains **fashion products.**
```

> You can find the records, settings and rules here:

Reviewer: ```suggestion
You can find the records, settings and [Rules](https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/) here:
```

- [`products.json`](./products.json)
- [`products_settings.json`](./products_settings.json)
- [`products_rules.json`](./products_rules.json)

> #### Product query suggestions index

Reviewer: ```suggestion
#### Query Suggestions index
```

> This index contains **query suggestions for products**.

Reviewer: ```suggestion
This index contains **Query Suggestions for products.**
```

You can find the records and settings here:

- [`products_query_suggestions.json`](./products_query_suggestions.json)
- [`products_query_suggestions_settings.json`](./products_query_suggestions_settings.json)

#### Articles index

> This index contains fake **articles** related to some product brands.

Reviewer: ```suggestion
This index contains **articles related to some product brands.**
```

You can find the records in the [`articles.json` file](./articles.json).

#### FAQ index

> This index contains fake **FAQ**.

Reviewer: ```suggestion
This index contains **frequently asked questions.**
```

You can find the records in the [`faq.json` file](./faq.json).

> ## Credentials

Reviewer 2: ```suggestion
## Import to you AlgoliaApp
Please refer to the guide [Exporting and Importing Algolia Indices](https://www.algolia.com/doc/guides/sending-and-managing-data/manage-your-indices/how-to/export-an-algolia-index/) in order to learn how to import the data into your Algolia application.

## Credentials
```

> If you want to use the dataset directly, without pushing it your application,
> just use the following credentials:

Reviewer: ```suggestion
If you want to use the dataset directly without pushing it to your own application, you can use the following credentials:
```

> Application ID: `latency`  
> API Key: `6be0576ff61c053d5f9a3225e2a90f76`

Reviewer: ```suggestion
- Application ID: `latency`  
- API key: `6be0576ff61c053d5f9a3225e2a90f76`
```

> ## Import to you Algolia app

Reviewer: ```suggestion
## Import to your Algolia app
```

Author: Oops, good catch!
